app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -179,10 +181,9 @@ class Core(
      * The [BrowserStore] holds the global [BrowserState].
      */
     val store by lazyMonitored {
-        BrowserStore(
-            middleware = listOf(
+        var middlewareList =
```

Comment:
nit: This is a `var` but you you do seem to manipulate this reference?

Reply:
Oops, my original implementation needed var.  But it should be val now.  I'll fix, thanks,

Reply:
I'll use a mutable list as suggested.